core/src/Cuda/Kokkos_CudaSpace.cpp

```diff
@@ -51,6 +51,15 @@
 #include <algorithm>
 #include <atomic>
 
+// The purpose of the following variable is to allow a state-based choice
+// for pinning UVM allocations to the CPU. For now this is considered
+// an experimental debugging capability - with the potential to work around
+// some CUDA issues.
+#ifdef KOKKOS_IMPL_DEBUG_CUDA_PIN_UM_TO_HOST
```

Comment:
Where is the option to exposed to users that sets `KOKKOS_IMPL_DEBUG_CUDA_PIN_UM_TO_HOST`?

Reply:
it is not right now. Matt would be ok by having to define this on the cxxflags. I am unsure whether we want an official external option, and whether we want to have more churn on cmake right now (though I suppose the change would be small).